src/main/java/arrays/Main.java

```diff
@@ -0,0 +1,78 @@
+        System.out.println("\nOriginal array to find a sum of positive elements is:");
+        System.out.println(Arrays.toString(numbersToSum));
+        System.out.println("The sum of all positive numbers is:");
+        System.out.println(sumOfArrayElements(numbersToSum, true));
+
+        int[] numbersToFindAverage = {132, 15, 67, 22};
+        System.out.println("\nOriginal array to find an average value is:");
+        System.out.println(Arrays.toString(numbersToFindAverage));
+        System.out.println("The average value of all numbers is:");
+        System.out.println((float) sumOfArrayElements(numbersToSum, false) / numbersToSum.length);
+
+        ArrayList<String> numbersArrayList = new ArrayList<>(Arrays.asList("One", "Two", "Tree", "Four"));
+        System.out.println("\nOriginal ArrayList is:");
+        System.out.println(numbersArrayList);
+        numbersArrayList.set(numbersArrayList.indexOf("Tree"), "Three");
+        System.out.println("Corrected ArrayList is:");
+        System.out.println(numbersArrayList);
+
+        ArrayList<Integer> newNumbersArrayList = new ArrayList<>(Arrays.asList(3, 9, 11, 18, 20, 22));
+        System.out.println("\nOriginal ArrayList is:");
+        System.out.println(newNumbersArrayList);
+        removeUnnecessaryElements(newNumbersArrayList);
+        System.out.println("Updated ArrayList (removed all numbers that are divisible by 3) is:");
+        System.out.println(newNumbersArrayList);
+
+        String objMethods = "Here is a method summary for Object in Java:";
+        objMethods += "\n-> clone()\n-> equals(Object obj)\n-> finalize()\n-> getClass()\n-> hashCode()\n-> notify()";
+        objMethods += "\n-> notifyAll()\n-> toString()\n-> wait()\n-> wait(long timeout)\n-> wait(long timeout, int nanos)";
+        System.out.printf("\n%s", objMethods);
+
+        System.out.println("\n\nArrays in Java have fixed size because they are implemented as a contiguous block of"
+            + " memory and need specified amount of memory to be reserved when it's initialized.\nBecause of"
+            + " that JVM can directly access needed element of array by calculating in the easy form the exact"
+            + " memory address where it's stored (using memory address of the first element, using the size of the"
+            + " declared type of data stored in this array and multiplying by the index of the element).\n"
+            + "It is quicker than the same operation for ArrayList or LinkedList. ");
+    }
+
+    static Integer[] convertArray(int[] oldArray) {
+        Integer[] newArray = new Integer[oldArray.length];
+        for (int i = 0; i < oldArray.length; i++) {
+            newArray[i] = oldArray[i];
+        }
+        return newArray;
+    }
+
+    static int sumOfArrayElements(int[] array, boolean onlyPositives) {
+        int result = 0;
+        for (int el : array) {
+            if (onlyPositives && el <= 0) {
+                continue;
```

Comment:
please never use break/continue/goto (in c# for instance) in your coding.

Reply:
Why not? Is there a recommendation not to use this (or other) operators?

Reply:
@Maks-Chornyi used another construction inside of the loop